src/hostnet/lib/dns_forward.mli

```diff
@@ -6,3 +6,12 @@ module Make
     (Time: V1_LWT.TIME) : sig
   val input: secondary:bool -> ip:Ip.t -> udp:Udp.t -> src:Ipaddr.V4.t -> dst:Ipaddr.V4.t -> src_port:int -> Cstruct.t -> unit Lwt.t
 end
+
+val only_ipv4: (Ipaddr.t * int) list -> (Ipaddr.t * int) list
+(** Since we don't currently support UDP to IPv6, filter all IPv6 nameservers
+    out of the list. *)
```

Comment:
does this need to exposed in the interface? It doesn't appear to be used outside of Dns_forward